clippy_lints/src/utils/conf.rs

```diff
@@ -561,6 +561,11 @@ define_Conf! {
     /// Which crates to allow absolute paths from
     (absolute_paths_allowed_crates: rustc_data_structures::fx::FxHashSet<String> =
         rustc_data_structures::fx::FxHashSet::default()),
+    /// Lint: PATH_ENDS_WITH_EXT.
+    ///
+    /// Additional file extensions to allow
+    (allowed_file_extensions: rustc_data_structures::fx::FxHashSet<String> =
+        rustc_data_structures::fx::FxHashSet::default()),
```

Comment:
I think it's worth re-wording this, since they're not file extensions if they're added to this config option, they're files or directories